src/lib/server/twilio.ts

```diff
@@ -94,12 +94,24 @@ const msgToSend = async (
 			msg = `Thanks for subscribing to reminders and friend availability notifications from ${url}! You can disable this at any time on your Profile page or by responding STOP.`;
 			break;
 		}
+		case 'reminder': {
+			const { phone } = msgComps;
+			msg = `Hi! It's your periodic reminder to update your schedule: https://playdate.help/login/${phone}`;
```

Comment:
We should probable provide a full login link with a token, unless you can think of a reason not to?

Reply:
mmm yeah we can do that. the only downside is the session could expire between when the reminder is sent and when the user finally clicks on the link but i think that's fine